exercises/two-fer/two-fer.js

```diff
@@ -0,0 +1,10 @@
+//
+// This is only a SKELETON file for the 'Two fer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export class TwoFer {
+  return() {
```

Comment:
The [spec](https://github.com/mluisamc/javascript/blob/stub-files/exercises/two-fer/two-fer.spec.js) does not expect a `class`, nor is this valid syntax.

Reply:
Ok, modified!

Reply:
I always wonder why `export const xyz` is valid but `export class xyz` is not 🤷‍♂

Reply:
It is valid, I was talking about the `return() {`